pipenv/cli.py

```diff
@@ -853,28 +835,24 @@ def update(
                     err=True,
                 )
                 sys.exit(1)
-        ensure_lockfile(keep_outdated=project.lockfile_exists, pypi_mirror=pypi_mirror)
-        # Install the dependencies.
-        do_install(
-            package_name=package,
-            more_packages=more_packages,
-            dev=dev,
-            three=three,
-            python=python,
-            pypi_mirror=pypi_mirror,
-            system=system,
-            lock=True,
-            ignore_pipfile=False,
-            skip_lock=False,
-            verbose=verbose,
-            requirements=False,
-            sequential=sequential,
-            pre=pre,
-            code=False,
-            deploy=False,
-            keep_outdated=True,
-            selective_upgrade=True,
-        )
+    do_lock(
+        verbose=verbose, clear=clear, pre=pre, keep_outdated=keep_outdated, pypi_mirror=pypi_mirror
+    )
+    do_sync(
+        ctx=ctx,
+        install=install,
```

Comment:
It is a good time to clean up arguments of `sync` now? (#2211 but maybe less drastic, just remove those that are really unused). My eyes roll every time I see it.

Reply:
i only cleaned up a few -- some i think we want to implement